src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -1207,8 +1207,10 @@ void ExportMusicXml::calcDivisions()
                 // special number 0 -> don’t show staff number in
                 // xml output (because there is only one staff)
 
+                /*
                 int sstaff = (staves > 1) ? st - strack + VOICES : 0;
                 sstaff /= VOICES;
+                */
```

Comment:
Here we could as well just remove those lines?
@lvinken ?

Reply:
Yes, of course. Text search shows variable sstaff is not used in that function. Probably left in place by accident after copying some code.